tests/jballerina-unit-test/src/test/resources/test-src/expressions/builtinoperations/clone-operation.bal

```diff
@@ -384,27 +385,24 @@ public function testCloneArrayWithError() returns boolean {
 
 public function testCloneMapWithError() returns boolean {
     map<error> errMap = {
-        e1: err1,
-        e2: err2,
-        e3: err3,
-        e4: err4
+        e1: error("h")
     };
 
-    map<anydata> ma = {
+    map<value:Cloneable> ma = {
         one: 1,
         two: "two",
         errMap: errMap
     };
-    map<error> errMapFromValue = <map<error>> ma["errMap"];
+    map<error> errMapFromValue = <map<error>> check ma["errMap"];
```

Comment:
This should be `checkpanic` right? There seems to be a bug here with `value:Cloneable`.

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/28081.